index.html

```diff
@@ -164,8 +167,8 @@ <h2 class="mb-4">Organization Team</h2>
                     </span>
                   </div>
                   <div class="text">
-                    <p class="name">Dennis Green</p>
-                    <span class="position">Marketing Manager</span>
+                    <p class="name">Michael Wu</p>
+                    <span class="position">Lazy web designer</span>
```

Comment:
Goddamn Ramesh, that's some serious fucking shade

Reply:
Hey, it's true. Otherwise we wouldn't be using a template and we would actually write the website. We could use boostrap to make it look good at the end.

Reply:
```suggestion
                    <span class="position">Professional and master web designer</span>
```


Want this?

Reply:
I don't care; it was just a joke 😛